### ISSUE TITLE
PoseTrack18 AP=38.5 and provided pose download link only has 74 json files? 

### ISSUE THREAD

**Issue by yw155** [mks0601/TF-SimpleHumanPose#33]
Hi @mks0601, I use provided model of 256x192_resnet50 and human detection results to test the accuracy on the PoseTrack 2018 dataset. The result is as follows:

Loading data
gt frames  : 8923
pred frames: 8923
Evaluation of per-frame multi-person pose estimation
saving results to ./out/total_AP_metrics.json
Average Precision (AP) metric:
& Head & Shou & Elb  & Wri  & Hip  & Knee & Ankl & Total\\
& 39.1 & 41.3 & 39.6 & 35.6 & 38.5 & 38.9 & 36.5 & 38.5 \

Furthermore, I downloaded the pose results from the provided link(https://cv.snu.ac.kr/research/TF-SimpleHumanPose/PoseTrack/pose_result/person_keypoints_256x192_resnet50_val_results.zip) and the number of included json files is only 74 while the number of groundtruth of val set is 170. The evaluation cannot be finished. I am not sure where is the problem. Could you give some suggestions? Thanks in advance.

**Comment by mks0601** [comment]
Hi, I checked that the version of posetrack18 is different from my git repo result and current version. My label version is posetrack18_v0.2_public_labels, however,  current one is v0.45. I think you have to train a new model with my pre-trained model on the COCO dataset following training details in README.

**Comment by yw155** [comment]
@mks0601, thanks for your reply and the suggestion. Could you share your label file v0.2 with me? The previous version cannot be downloaded. I would like to take a test first. Thank you.

**Comment by mks0601** [comment]
Sure. link[https://cv.snu.ac.kr/research/TF-SimpleHumanPose/PoseTrack/posetrack18_v0.2_public_labels.tar.gz]

**Comment by yw155** [comment]
@mks0601, thank you very much.

**Comment by yw155** [comment]
Hi @mks0601, using the previous labels can get the AP of 70.4. And, I would like to know which human detection method is used to generate the human_detection_val.json?

**Comment by mks0601** [comment]
I used [Detectron](https://github.com/facebookresearch/Detectron), which is Mask R-CNN

**Comment by yw155** [comment]
@mks0601, thank you for telling me.

**Comment by wmcnally** [comment]
@yw155 did you get an AP of 70.4 using detected bounding boxes or ground-truth? Thanks!
